src/style.css

```diff
@@ -0,0 +1,111 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.searchBig {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  width: 100%;
+  font-size: 16px;
+  line-height: 66px;
+  padding-left: 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-weight: 700;
+}
```

Comment:
most of the styles are the same for searchBig and searchSmall, create a common class (input for example) and them add second class smallInput for smaller one, where you will rewrite some of the styles (like bigger font, etc)